src/panels/config/backup/components/config/ha-backup-config-data.ts

```diff
@@ -75,8 +76,11 @@ class HaBackupConfigData extends LitElement {
 
   @state() private _showAddons = false;
 
+  @state() private _dbCanbeIncluded = true;
```

Comment:
```suggestion
  @state() private _showDbOption = true;
```

And rename `_checkDbCanBeIncluded` to `_checkDbOption`?